### ISSUE TITLE
#4: Add to reflection.md per milestone3 instructions

### ISSUE THREAD

**Issue by moniquewong** [UBC-MDS/DSCI_532_group-106_supermarket#52]
**Relevant milestone3 instructions**:
Large-scale changes you make since milestone2 must be documented in your reflection.md file and you should link to particular issues in your repos whenever possible. You must also add some justification for why you prioritized certain bugs/features over others and what your overall strategy was for maintenance. You should try to come up with a plan for this early in the week and then make adjustments/modifications as you learn more information and get additional feedback from user testing.

**The reflection.md file you will submit will include several elements and link to issues when appropriate. Below is a list to help you get started.**

- Start with the reflection.md file you included with milestone2
- Make any necessary edits or changes (for e.g., if you indicated in your milestone2 reflection that your app had a bug and you fixed it in week 3, you should update your reflections file)
- Wishlist features/bug-fixes
- If there is anything you wish you had more time to implement or fix, you can list it here.
- Include a brief, high-level summary of any changes you decided to make
- Provide a brief, high-level summary of the feedback you received during the lab session (details below)
- Reflect on the feedback you received (more guidance is below)

**Reflection on the usefulness of the feedback you received**
@moniquewong - complete by Thursday evening
Here are some questions to get some ideas flowing. You aren't expected to (and probably shouldn't) answer all of these questions.

- Reflect on the usability of your app. How easy or hard was it for your peers to use your app? Did you expect them to use the app in they way they did?
- What things did you hear that's similar across reviewers? Is there a theme here?
- From the feedback, what things do you think are appropriate to change in your app, particularly given the time frame you have to improve the app?
- What is unreasonable to change, and why? Time restraints? Too technical?
- What feedback was the most/least valuable?
- What part of the feedback process was most/least valuable?
- What did you learn from your experience being a "fly-on-the-wall". Was this useful? Why or why not?
- Did the feedback process lead to an improved app? What parts were valuable? What parts were not valuable?

**Comment by moniquewong** [comment]
Strategy:
- Least effort for most benefit in user experience

Capturing adjustments made so far:
- Adding comparison feature
- Adding instructions and guiding examples

**Comment by reikookamoto** [comment]
Grammatical errors:
- "As each plot loaded with different store or day of week / time of day selections, the colour scale (heat map) and y-axis scale (bar plots) updated to the unique values of the filtered selection." I think a word is missing before the first comma
- "While being able to compare between different day of week and time of day selections with a store has merit, we prioritized adding functionality to make that possible." I'm not sure if I'd use "while" here. Try "since" instead?

General comments:
- We wanted to change Times New Roman to a sanf-serif font but we didn't have time to do this
- Changes made to the directory structure and files inside: changed file names so it's easy to understand what's inside, `app.py` written in PEP-8 style with proper docstrings. This doesn't directly improve the user's experience but now that the project is organized and ordered, it'll be easier for us to go back and make changes later on, if necessary. Although re-factoring was suggested, we decided it wasn't our priority to make these changes.
